### ISSUE TITLE
ignore syntax doesn't appear to work

### ISSUE THREAD

**Issue by ralyodio** [lukeed/taskr#133]
```
const paths = {
  scripts: ["client/**/*.js", "!client/app.js"]
};

export default function* () {
  yield this.watch(paths.scripts, "build")
}

export function* build () {
  yield this
    .source(paths.scripts)
    .concat('app.js')
    .target("client");
}
```

I'm getting multiple copies of the code in app.js, one for each time the build task is executed

**Comment by lukeed** [comment]
Hi chovy,

This was actually reported earlier (#106) and it's on the radar for the near future! Heck, I might even get to it tonight, seeing that I've tip-toed around the same issue in previous projects :wink:

**Comment by lukeed** [comment]
@chovy I forgot that I had actually fixed this for myself a few weeks ago. Just sent over the commits & then I'll ping you when they've made it in :+1:

**Comment by lukeed** [comment]
Closed in 0.8.7. Please let me know if you are still having issues. :)
